README.md

```diff
@@ -248,6 +248,8 @@ myData :=  map[string]interface{}{
 	]
 }
 ```
+Note:
+When producing to a station with more than one partition, the producer will produce messages in Round Robin to the partitions.
```

Comment:
... the producer will produce messages in a Round Robin fashion between the different partitions.